test/integration/singlecluster/webhook/jobs/appwrapper_webhook_test.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jobs
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	awv1beta2 "github.com/project-codeflare/appwrapper/api/v1beta2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/discovery"
+
+	"sigs.k8s.io/kueue/pkg/controller/jobframework"
+	"sigs.k8s.io/kueue/pkg/controller/jobs/appwrapper"
+	"sigs.k8s.io/kueue/pkg/util/kubeversion"
+	"sigs.k8s.io/kueue/pkg/util/testing"
+	testingaw "sigs.k8s.io/kueue/pkg/util/testingjobs/appwrapper"
+	"sigs.k8s.io/kueue/test/util"
+)
+
+var _ = ginkgo.Describe("AppWrapper Webhook With manageJobsWithoutQueueName enabled", ginkgo.Ordered, func() {
+	var ns *corev1.Namespace
+	ginkgo.BeforeAll(func() {
+		discoveryClient, err := discovery.NewDiscoveryClientForConfig(cfg)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		serverVersionFetcher = kubeversion.NewServerVersionFetcher(discoveryClient)
+		err = serverVersionFetcher.FetchServerVersion()
```

Comment:
nit: is this relevant? If then then let's drop it.

Reply:
removed.  I started from the job_webhook and didn't think carefully enough about what I actually needed.